src/main/java/com/solopov/urlshorter/web/dto/UrlGenerateRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package com.solopov.urlshorter.web.dto;
+
+import com.solopov.urlshorter.dao.entities.ShortenedUrl;
+
+import javax.validation.constraints.NotBlank;
+
+public record UrlGenerateRequestDto(@NotBlank String url, @NotBlank String title) {
+
+    public ShortenedUrl convert() {
```

Comment:
conversion between DTO <-> entities usually delegated to a separate converter/mapper

Reply:
++ will change

Reply:
upd coommit: implemented